requirements-tests.txt

```diff
@@ -1,5 +1,6 @@
 ply
-pytest
+pytest==2.9.1
```

Comment:
On linux + python3.4, CI would mysteriously install `pytest` at 2.6.x ... which didn't work with `pytest-timeout` ... I should probably remove this `pytest-timeout` dependendy.